src/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { FilterOptions } from './types/FilterOptions';
+import { FilterType } from './types/FilterType';
+import { Todo } from './types/Todo';
+
+export function filterTodos(
+  todos: Todo[],
+  { query, filterType }: FilterOptions,
+) {
+  let todoToRecieve = [...todos].filter(({ completed }) => {
```

Comment:
```suggestion
  let todoToRecieve = todos.filter(({ completed }) => {
```
`filter` returns a new array, you don't need to create a copy, also you don't need to filter if the `filterType` is `FilterType.Completed`